SUMMARY.md

```diff
@@ -134,6 +134,7 @@
 		* [POST Update Call](voice/methods/calls/postCallsCallId.md)
 		* [PUT Pause/Resume Recording](voice/methods/recordings/putCallsCallIdRecording.md)
 	* [Conferences](voice/methods/conferences/about.md)
+		* [GET Account's Conferences](voice/methods/conferences/getConferences.md)
```

Comment:
In recording, we use `Account Recordings` (no apostrophe). I'm no English expert so I'm not sure which one is best.